tidy3d/web/api/webapi.py

```diff
@@ -926,6 +927,35 @@ def real_cost(task_id: str, verbose=True) -> float:
     return flex_unit
 
 
+@wait_for_connection
+def account(verbose=True) -> Account:
+    account_info = Account.get()
+    if verbose and account_info:
+        console = get_logging_console()
+        credit = account_info.credit
+        credit_expiration = account_info.credit_expiration
+        cycle_type = account_info.allowance_cycle_type
+        cycle_amount = account_info.allowance_current_cycle_amount
+        cycle_end_date = account_info.allowance_current_cycle_end_date
+        free_simulation_counts = account_info.daily_free_simulation_counts
+
+        message = ""
+        if credit:
```

Comment:
Let's check `if credit is not None` instead. If the credit is 0 we still want to print that it is 0, while this will not happen currently.